modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -32,11 +32,29 @@ const moduleAvailable = (name: string) => {
 
 const webpackRemap: Array<Record<string, any>> = moduleAvailable('@ephox/swag') ? [
   {
-    test: /\.js|\.tsx?$/,
+    test: /\.js|\.mjs|\.tsx?$/,
     use: [ '@ephox/swag/webpack/remapper' ]
   }
 ] : [];
 
+const webpackSharedRules = webpackRemap.concat([
+  {
+    test: /\.mjs$/,
+    type: 'javascript/auto',
+    use: []
+  },
+  {
+    test: /\.js|\.mjs$/,
+    use: [ 'source-map-loader' ],
+    enforce: 'pre'
+  },
+
```

Comment:
Is there a reason we have a new line here, but not above?

Reply:
Either an oversight or a copy/paste error. I'll fix it.